### ISSUE TITLE
include instructions to reset screen reader preferences?

### ISSUE THREAD

**Issue by robfentress** [w3c/aria-at#172]
Should there be explicit instructions somewhere to [reset one's screen reader preferences](https://www.apple.com/voiceover/info/guide/_1136.html#vo27926) to their defaults before starting a test plan run?  Otherwise, I worry there may be inconsistent or difficult-to-interpret results because a Tester has their verbosity settings set a certain way or perhaps the last time they used VoiceOver they had [switched to Group Mode](https://www.apple.com/voiceover/info/guide/_1134.html#mchlp2711) and forgot to switch back to DOM Mode--things like that.

**Comment by zcorpan** [comment]
Yes; this is a duplicate of https://github.com/w3c/aria-at/issues/113
